### ISSUE TITLE
Spawners dupe on break

### ISSUE THREAD

**Issue by ImHumbug** [dannyferguson/MineableSpawners#85]
It seems like the plugin always transfer a spawner on broke to the player inventory even if the player have no permission.
Example 1: Player with no permission try to break a spawner with a pickaxe that met the conditions to break spawners. Result: Spawner won't break and gives the no permission message but also the plugin transfer a copy of the spawner to the player inv called &dGenerator.
Example 2: Player with permission try to break a spawner with a pickaxe that met the conditions to break spawners. Result: Spawner broke but it also give me another copy of the same so I've got 2, the one I broke and other one called &dGenerator.

if you try to break the spawner with a pickaxe that doesn't meet the conditions, this doesn't happen. I'm using latest version on 1.18.

**Comment by ImHumbug** [comment]
Ok, I did more testing, so I'll be more specific. Tried as an OP, and I always get an extra spawner when I break one using a tool that is in the tools list (doesn't matter if it have silk touch or not), the extra one is directly dropped to my inv even if that option is disabled and also send the message: You succesfully mine a %mob% Spawner, the item is applied to your inventory. If I try to break the spawner with a tool that is not on the list, the spawner break properly, and I just receive one copy to my inv, no chat message. 

Without OP and without any permission of the plugin if I try to break the spawner with a tool that is not on the tool list, the spawner won't break, and it won't send a copy to my inv, if I try with a tool that is on the list but without silk touch the result is the same. If I try the same with a tool that is on the tools list and with silk touch then the spawner won't break with the no permission message, but it stills send me a copy directly to my inv.

mining:
  # Whether or not to require the permission "mineablespawners.mine" to mine a spawner.
  require-permission: true
  # Whether or not to require the permission "mineablespawners.mine.<type>" to mine a spawner.
  require-individual-permission: false
  # Which tools should be used to mine spawners

this is my config:
  tools:
    - 'DIAMOND_PICKAXE'
    - 'NETHERITE_PICKAXE'
    - 'IRON_PICKAXE'
    - 'GOLDEN_PICKAXE'
    - 'STONE_PICKAXE'
    - 'WOODEN_PICKAXE'
  # Whether or not to require silk touch to mine a spawner.
  require-silktouch: false
  # Should a different of silk touch be required? "required-level" has no effect if "require-silktouch-level" is false.
  require-silktouch-level: false
  required-level: 2
  # The chance it will drop in percentage from 0-100
  chance: 100

also I have drop-to-inventory: false

**Comment by FireBug7191** [comment]
I am having the same issue did you ever figure it out?

**Comment by matipoirierg** [comment]
> I am having the same issue did you ever figure it out?

Issue happens when you have EconomyShopGUI

**Comment by shortybsd** [comment]
> > I am having the same issue did you ever figure it out?
> 
> Issue happens when you have EconomyShopGUI

Yup, this mod is causing it! We need  to report to that mod author.
